src/utils/find-files.test.ts

```diff
@@ -0,0 +1,103 @@
+jest.mock('glob')
+jest.mock('path')
+jest.mock('./read-file')
+import glob from 'glob'
+import path from 'path'
+import { getFileExtensionPattern } from './read-file'
+const mockedGlob = glob as jest.Mocked<typeof glob>
+const mockedPath = path as jest.Mocked<typeof path>
+const mockedGetFileExtensionPattern = getFileExtensionPattern as jest.MockedFunction<
+  typeof getFileExtensionPattern
+>
+
+const mockedFileNames = ['/dev/config/development.yml', 'schema.json']
+const mockedResolvedPath = '/dev'
+mockedGlob.sync = jest.fn().mockReturnValue(mockedFileNames)
+mockedPath.resolve = jest.fn().mockReturnValue(mockedResolvedPath)
+mockedGetFileExtensionPattern.mockReturnValue('json')
+
+import { findConfigFiles, findFiles, isNotSchema } from './find-files'
+
+describe('findFiles works as expected', () => {
```

Comment:
nit: "works as expected" sounds very redunant IMO. every test essentially is about checking whether code works as expected.

IMO we could just use the function names for these util `describe` statements.

```suggestion
describe('findFiles()', () => {
```

Reply:
fine with me

Reply:
making this comment once is sufficient ;)

Reply:
updated